docs/2.8.0/docs/getting-started/manual-download.rst

```diff
@@ -44,13 +52,13 @@ To do that:
 
    Note: This warning means you have not told gnupg that you trust this key actually belongs to Digital Asset. The ``[unknown]`` tag next to the key has the same meaning: ``gpg`` relies on a web of trust, and you have not told it how far you trust this key. Nevertheless, at this point you have verified that this is indeed the key that has been used to sign the archive.
 
-5. The next step is to extract the tarball and run the install script (unless you chose the Windows installer, in which case the next step is to double-click it)::
+6. The next step is to extract the tarball and run the install script (unless you chose the Windows installer, in which case the next step is to double-click it). You will need to replace the phrase ``PART_TWO_VERSION_HERE`` in the following program with version you obtained in Part 2. **Do not** use the internal version mentioned in the note in Part 3.::
```

Comment:
```suggestion
6. The next step is to extract the tarball and run the install script (unless you chose the Windows installer, in which case the next step is to double-click it). You will need to replace the phrase ``STEP_TWO_VERSION_HERE`` in the following program with version you obtained in step 2. **Do not** use the internal version mentioned in the note in step 3.::
```

Reply:
Correct me if I'm misunderstanding, but I think "part" should be "step" in all instances. (At first I was looking for a section 2 or other large part of the document, not a numbered step.)